faqs.html

```diff
@@ -5,29 +5,41 @@
 <div class="container white-text">
   <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
   <ul class="collapsible black">
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <div class="card-content"> <span class="card-title clem"><i class="material-icons">error_outline</i></span>
+        <p>As passionate we are about cybersecurity, safety is our number one priority. We sincerly hoped that the situation around COVID-19 would improve enough to continue our event on Clemson University’s campus. However, based on local health experts and community leader recommendations, it has been determined that <strong>CUCTF 2020 will be a completely virtual event on October 3rd. </strong> We refuse to be setback from this announcement, and will use this oppertunity to: </p>
+            <ul class="browser-default">
+                <li>Allow more universities to participate by increasing our CTF infrastructure </li>
+                <li>Present opening and closing ceremonies via Zoom</li>
+                <li>Utilize Slack or Discord to answer questions, push notifications, and maintain a connection with participants</li>
+            </ul></div></div>
+    </li>
     <li>
       <div class="collapsible-header black clemson-orange-text">Where is the event being held?</div>
-      <div class="collapsible-body"><span>The even is being held at Clemson University's Watt Family Innovation Center. The address is <a rel="external noopener" href="https://goo.gl/maps/XZTw4u9Mu72V6P7s6" target="_blank">405 S Palmetto Blvd, Clemson, SC 29634</a>.</span></div>
+      <div class="collapsible-body"><span>CUCTF 2020 is being held online.</span></div>
     </li>
     <li>
       <div class="collapsible-header black clemson-orange-text">When is the event?</div>
-      <div class="collapsible-body"><span>CUCTF is taking place October 3rd, 2020.</span></div>
+      <div class="collapsible-body"><span>CUCTF 2020 is taking place October 3rd, 2020.</span></div>
     </li>
+
     <li>
       <div class="collapsible-header black clemson-orange-text">Who is eligible for the event?</div>
-      <div class="collapsible-body"><span>Only university and college teams are eligible for CUCTF.</span></div>
+      <div class="collapsible-body"><span>Only university and college teams are eligible for CUCTF 2020.</span></div>
     </li>
     <li>
       <div class="collapsible-header black clemson-orange-text">How many people are allowed on a team?</div>
-      <div class="collapsible-body"><span>You may have up to 4 members on a competing team and, optionally, up to 2 members for the CTF Bootcamp.</span></div>
+      <div class="collapsible-body"><span>Team sizes do not matter for CUCTF 2020, any amount of university students can join a team.</span></div>
```

Comment:
We might want to clarify that this is because the competition will be remote. Maybe "Because CUCTF 2020 will be a remote competition, team sizes are not limited and any number of students can join a team."

Reply:
I'd probably still like this change or at least some kind of rewording acknowledging this is because of the move to remote as I don't want people to expect unlimited team sizes next year (assuming we are) in person as physical space limits the size of teams.

Reply:
`Under normal conditions team size is limited to 4 students. However, because CUCTF 2020 is a remote event, team size does not matter for CUCTF 2020. Any amount of students from the same university can join a team.`